pkg/gofr/gofr_test.go

```diff
@@ -592,19 +582,51 @@ func TestStaticHandler(t *testing.T) {
 		}
 
 		if tc.expectedBodyLength != 0 {
-			contentLength, _ := strconv.Atoi(resp.Header.Get("Content-Length"))
-			assert.Equal(t, tc.expectedBodyLength, contentLength, "TEST [%d], Failed at Content-Length.\n %s", it, tc.desc)
+			contentLength := resp.Header.Get("Content-Length")
+			assert.Equal(t, strconv.Itoa(tc.expectedBodyLength), contentLength, "TEST [%d], Failed at Content-Length.\n %s", it, tc.desc)
 		}
 
 		if tc.expectedResponseHeaderType != "" {
 			assert.Equal(t,
 				tc.expectedResponseHeaderType,
 				resp.Header.Get("Content-Type"),
-				"TEST [%d], Failed at Expected Content-Type.\n%s",
-				it,
-				tc.desc)
+				"TEST [%d], Failed at Expected Content-Type.\n%s", it, tc.desc)
 		}
 
 		resp.Body.Close()
 	}
 }
+
+func createPublicDirectory(t *testing.T, htmlContent []byte) error {
+	directory := "./" + publicDir
+	if _, err := os.Stat(directory); err != nil {
+		if err := os.Mkdir("./"+publicDir, 0755); err != nil {
+			t.Errorf("Couldn't create a "+publicDir+" directory, error: %s", err)
```

Comment:
you are using Errorf

so it should be 

			t.Errorf("Couldn't create %s directory, error: %s", publicDir, err)